### ISSUE TITLE
Decide the fate of mod

### ISSUE THREAD

**Issue by vborovin** [sm-augmented/hub#5]

**Comment by vborovin** [comment]
Lupus:
If gameplay changes are not going to be made at all, from a non-technically standpoint I'd love to see any armor or weapons that can be finished added. Other than that, once the mod is returned fully to what it was, and any bug fixes that can be made are made, if you have no intention of either finding a new lead or just making no effort to make any further gameplay improvements or changes, then just leave it once it's there.

**Comment by SteelDad40k** [comment]
I think it's worth asking the question, what exactly do we want to see in SMA. New armor, weapons or maybe a new unique mode with unique classes?

**Comment by vborovin** [comment]
> I think it's worth asking the question, what exactly do we want to see in SMA. New armor, weapons or maybe a new unique mode with unique classes?

Yeah, what is the goal of mod now? Do we still keep all those plans we had before, even without the person who was responsible for designing them?

**Comment by SteelDad40k** [comment]
Unfortunately, I don’t know what the plans were regarding the mod, but I think it’s worth implementing what was known, and then already thinking about what will happen next.

**Comment by vborovin** [comment]
> Unfortunately, I don’t know what the plans were regarding the mod, but I think it’s worth implementing what was known, and then already thinking about what will happen next.

There were plans for a very long shot. We may keep them still, but details are to be made from scratch. These plans included:

- Focus on PvE mostly with PvP adapted to PvE basis given free time
- 7 new classes
- Numerous new weapons and perks, finishing changes for existing weapons and perks to align them into one system, including but not limited to:
  - 2H/DW versions of melee weapons
  - 3 shields
  - Phobos and Tigrus versions of BP and StoB
  - New equipment
  - God-dedicated perks for chaos and their mirrors for loyals
- AI improvements
- Numerous QoL changes, including UI and animation changes and fixes
- New maps' setups, including but not limited to:
  - Infinite horde mode
  - Revamped arenas, each lasts full 20 waves and dedicated to specific enemy side
- And more?
